rasa/cli/data.py

```diff
@@ -258,3 +270,63 @@ def _migrate_domain(args: argparse.Namespace) -> None:
     import rasa.core.migrate
 
     rasa.core.migrate.migrate_domain_format(args.domain, args.out)
+
+
+async def _convert_to_yaml(
```

Comment:
@souvikg10 Can we also get tests for this entry point? Or better even, the one above which comes from the CLI?